### ISSUE TITLE
Tagging an answer/choice

### ISSUE THREAD

**Issue by froggypaule** [ds-wizard/ds-wizard#262]
**The Question**
Tagging an answer: the idea is by tagging an answer, I allow only this option/answer to appear in the questionnaire.

Also, if one tags the follow up questions to an answer, here too, I expected this answer to be tagged.

Now, I know that questions and answers/choices are not the same entities, but is there a fundamental reason why this couldnt be possible?

**Additional context**
I already asked a similar question re tagging explanatory text in a question. 
The whole idea behind this thinking is being able to provide a KM which structurally can be specialised (by tags then) for say, research domains, data types, without have to create a new KM as a specialisation of the parent KM.
In other words, specialisation occurs in projects/project templates.

Any thought?

**Comment by MarekSuchanek** [comment]
Can you please also add this in https://ideas.ds-wizard.org as it seems like potential feature request that might be wanted also by others?

**Comment by froggypaule** [comment]
see https://ideas.ds-wizard.org/posts/128/tagging-answer-and-choices
